src/timer.rs

```diff
@@ -321,3 +372,369 @@ timer! {
     TIMER_A2, tim2,
     TIMER_A3, tim3
 }
+
+/// ********************************            TIMER 32            ********************************
```

Comment:
It might be a good idea to split this into a new file

Reply:
Indeed